### DISCUSSION TITLE
手机端网页图标

### DISCUSSION THREAD

**Discussion by happy2h** [usememos/memos#38] [Ideas]
目前next分支将网页添加到桌面没有icon的图标显示，希望可以添加。
例如`<link rel="apple-touch-icon" href="[static](https://so.csdn.net/so/search?q=static&spm=1001.2101.3001.7020)/my/img/favicon.ico"/>`这种

**Comment by boojack** [comment]
next 版本应该是没有手机样式的呀，目前打算把网页版做好。

**Reply by happy2h** [reply]
其实我的意思是在手机上将网页添加到主屏幕是，会在桌面添加一个图标，可以把这个图标也加到next版本中。
![Uploading D453B062-D6CA-4BBB-A54B-310300A811B1.png…]()

**Reply by happy2h** [reply]
![F3048CE8-066A-4FEC-98F2-639C6B3D0FFE](https://user-images.githubusercontent.com/29790236/158633599-63685dea-8d26-4605-a1c0-27c668f61cd1.png)

**Reply by boojack** [reply]
你可能用的是 latest 版，适配了手机样式。但是在 next 版中，我把手机样式移除了，准备清理代码，主打 web 端，所以这个图标加了也没用。等 web 端做完后，也可能把手机端样式加回来，可以到时候再加这个图标。

**Reply by happy2h** [reply]
好的
